content/en/blog/2024/collector-roadmap/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: The roadmap to v1 for the OpenTelemetry Collector
+linkTitle: Collector Roadmap
+date: 2024-05-02
+cSpell:ignore: Boten Broadbridge Hrabovcak Pantuza pushback Shishi Vijay
+author: '[Alex Boten](https://github.com/codeboten) (Honeycomb)'
+---
+
+The OpenTelemetry Collector is a very popular component in OpenTelemetry that
+has been under heavy development for quite some time. It is a binary that allows
+many formats of telemetry to be sent to it, transformed, and emitted to a
+destination. Much has been said about the Collector over the past few years in
+various blog posts and talks. Here's a small list of talks about the Collector
+if you haven't had the chance to learn about it:
+
+- [Connected Observability Pipelines in the OpenTelemetry Collector](https://www.youtube.com/watch?v=uPpZ23iu6kI)
+- [Telemetry Showdown: Fluent Bit Vs. OpenTelemetry Collector](https://www.youtube.com/watch?v=ykq1F_3PmJw)
+- [OpenTelemetry Collector Deployment Patterns](https://www.youtube.com/watch?v=WhRrwSHDBFs)
+
+The Collector has been a core component for organizations looking to adopt
+OpenTelemetry as part of their strategy to improve the telemetry emitted by
+their systems. Organizations around the world have already adopted it and
+successfully process large amounts of data through pipelines as documented by
+these various talks:
+
+- [Adopting OpenTelemetry Collector @ eBay - Swapping Engines Mid Flight - Vijay Samuel , eBay](https://www.youtube.com/watch?v=tZJd6W-CIcU)
+- [Ingesting 6.5 Tb of Telemetry Data Daily Through OpenTelemetry Protocol and Coll... Gustavo Pantuza](https://www.youtube.com/watch?v=aDysORX1zIs)
+- [Today, Not Tomorrow: Scalable Strategies for Migrating to Open... Jason Anderson & Kevin Broadbridge](https://www.youtube.com/watch?v=iPGd9_aYu-A)
+- [How and Why You Should Adopt and Expose OSS Interfaces Like OTel... - Daniel Hrabovcak & Shishi Chen](https://www.youtube.com/watch?v=D71fK2MFreI)
+- [Why, How to, and Issues: Tail-Based Sampling in the OpenTelemetry Collector - Reese Lee, New Relic](https://www.youtube.com/watch?v=l4PeclHKl7I)
```

Comment:
Can we have the full titles of those talks?

Reply:
Updated